Makefile.common

```diff
@@ -198,6 +198,11 @@ else
 endif
 endif
 
+.PHONY: common-yamllint
+common-yamllint:
+	@echo ">> running yamllint on all YAML files in the repository"
+	docker run --rm -v "${PWD}":/yaml sdesbure/yamllint find ./ -type f \( -iname \*.yml -o -iname \*.yaml \) | xargs yamllint --no-warnings
```

Comment:
We typically prefer to use local commands, as this won't won't work in our CI (CicleCI Docker in Docker won't volume map).

Reply:
As local command, this works fine without find/xargs.
```bash
yamllint .
```

If you just want to lint YAML files added to git:
```bash
git ls-files '*.yml' '*.yaml' | while read -r file ; do yamllint "$$file"; done
```